hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -298,6 +298,8 @@
     SCHEMA_REPOSITORY_HTTP_CONNECT_TIMEOUT_MS("schema.repository.http.connect.timeout.ms", 2000),
     SCHEMA_REPOSITORY_ONLINE_CHECK_PERMITS_PER_SECOND("schema.repository.online.check.permits.per.second", 100.0),
     SCHEMA_REPOSITORY_ONLINE_CHECK_ACQUIRE_WAIT_MS("schema.repository.online.check.acquire.wait.ms", 500),
+    SCHEMA_REPOSITORY_SUBJECT_SUFFIX_ENABLED("schema.repository.subjectSuffixEnabled", false),
```

Comment:
Hmmm... but wouldn't the dot way be a little bit awkward  in management module, which is built on Spring? Every dot means a nested configuration class. I think this is why you already have `serverUrl` property here not conforming to the style.

Reply:
Of course it would be awkward and my comment is not meant to change them. :)

My point is to change only the string properties in this file (`Configs` class in commons module) which is originally used by modules based on archaius (hermes-frontend, hermes-consumers).

Management module uses Spring and its own configuration/properties classes as we know. This class is used by the management only for the sake of referencing to default values of these properties, so they are the same in both places (like both frontends and management have `SCHEMA_REPOSITORY_SUBJECT_SUFFIX_ENABLED ` and `subjectSuffixEnabled` set to `false`). Properties introduced in management in this PR are completely fine. :)

Reply:
I have changed this, but I assume you know that the drawback is that the property keys are different now in management and consumers/frontend modules what might be confusing.

Reply:
Yes, I'm aware of that, it's fine. Thank you.